tests/classes/event_properties/test_event_property_definitions.py

```diff
@@ -336,3 +342,371 @@ def test_source_plate_uuid_errors(app, source_plates):
             source_plate_property.value
 
         assert len(source_plate_property.errors) > 0
+
+
+@pytest.mark.parametrize("run_id", [5])
```

Comment:
How do these work? Are the tests run each time a mark is made? Can they be made at the same time instead?

Reply:
It uses all marks at the same time but it does all combinations of all values. Because there is only one option for each mark this generates one single test.

Reply:
So does that mean it can be made into 1 line instead?

Reply:
Do you have an example?